db/migrate/20201129151451_create_ratings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRatings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :ratings do |t|
+      t.integer :article_id
+      t.integer :user_id
+      t.integer :rate
```

Comment:
maybe should rating be zero by default?
```suggestion
      t.integer :rate, default: 0, null: false
```